build.sbt

```diff
@@ -1,189 +1,115 @@
 import sbtcrossproject.{ CrossType, crossProject }
 
-ThisBuild / organization := "io.circe"
-
-val compilerOptions = Seq(
-  "-deprecation",
-  "-encoding",
-  "UTF-8",
-  "-feature",
-  "-language:existentials",
-  "-language:higherKinds",
-  "-unchecked",
-  "-Ywarn-dead-code",
-  "-Ywarn-numeric-widen",
-  "-Yrangepos"
-)
+val Scala212V = "2.12.15"
+val Scala213V = "2.13.7"
 
-val scala212 = "2.12.15"
-val scala213 = "2.13.7"
-
-val circeVersion = "0.14.2"
+val circeVersion = "0.14.3"
 val paradiseVersion = "2.1.1"
 
 val jawnVersion = "1.3.2"
 val munitVersion = "0.7.29"
 val disciplineMunitVersion = "1.0.9"
 
-val previousCirceGenericExtrasVersions = Set("0.14.0", "0.14.1")
-
-def priorTo2_13(scalaVersion: String): Boolean =
-  CrossVersion.partialVersion(scalaVersion) match {
-    case Some((2, minor)) if minor < 13 => true
-    case _                              => false
-  }
-
-ThisBuild / crossScalaVersions := Seq(scala212, scala213)
+ThisBuild / tlBaseVersion := "0.14"
+ThisBuild / tlCiReleaseTags := false
 
-val baseSettings = Seq(
-  scalacOptions ++= compilerOptions,
-  scalacOptions ++= (
-    if (priorTo2_13(scalaVersion.value))
-      Seq(
-        "-Xfuture",
-        "-Yno-adapted-args",
-        "-Ywarn-unused-import",
-        "-Ypartial-unification"
-      )
-    else
-      Seq(
-        "-Ymacro-annotations",
-        "-Ywarn-unused:imports"
+ThisBuild / organization := "io.circe"
+ThisBuild / crossScalaVersions := List(Scala212V, Scala213V)
+ThisBuild / scalaVersion := Scala213V
+
+ThisBuild / githubWorkflowJavaVersions := Seq("8", "11", "17").map(JavaSpec.temurin)
```

Comment:
I honestly think testing on all these JDKs is unnecessary.

Reply:
Maybe, but Http4s tests on these 3 as well.

Reply:
Yes, but that's because http4s is entangled with very JDK-specific stuff like TLS. See discussion in:
- https://github.com/http4s/sbt-http4s-org/issues/154

Reply:
Fair enough.  I can switch it to just testing 8 and 17